### ISSUE TITLE
conv8 output all negative

### ISSUE THREAD

**Issue by realshijy** [keunhong/pytorch-soundnet#4]
hi,
sry to bother
i'd like to know whether it's normal all the output features of conv8_objs are negative.

And also, i am not sure when extracting the features, is it necessary to plus 256 after liborosa.load and how to deal with batch norm

**Comment by realshijy** [comment]
i extracted feature of ESC-50 dataset
